usefulUtilities/botinator/tabletApp/js/botinator_ui.js

```diff
@@ -0,0 +1,214 @@
+/*
+
+    Botinator
+    Created by Milad Nazeri on 2019-01-07
+    Copyright 2019 High Fidelity, Inc.
+
+    Distributed under the Apache License, Version 2.0.
+    See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+    TABLET UI JS
+    
+*/
+
+
+// *************************************
+// START VARS AND CONSTS
+// *************************************
+// #region VARS AND CONSTS
+
+
+var loadingContainer;
+var contentBoundaryCorners1Position;
+var contentBoundaryCorners2Position;
+var volumeSlider;
+var totalNumberOfBotsNeeded;
+var availableACs;
+var playStopButton; 
+var updateBotDataButton;
+var playButtonDisabled = true;
```

Comment:
any reason you can't replace instances of this with `playStopButton.disabled` ?

Reply:
I completely forgot it's a getter/setter